docs/tutorials/ofo.md

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: 2
+id: ofo
+title: Cloud native flags with the OpenFeature Operator
+---
+
+# Cloud native feature-flagging with the OpenFeature Operator
+
+In the following tutorial, we'll see how to leverage _flagd_ and the OpenFeature Operator to enable cloud-native, self-hosted feature flags in your Kubernetes cluster. [flagd](https://github.com/open-feature/flagd) is a "feature flag daemon with a Unix philosophy". Put another way, it's a small, self-contained binary that evaluates feature flags, uses standard interfaces, and runs just about anywhere. It can be deployed in a central location serving multiple clients or embedded into a unit of deployment (such as a pod in Kubernetes). The [OpenFeature Operator](https://github.com/open-feature/open-feature-operator) is a K8s-flavored solution for easily adding flagd to any relevant workloads. It parses Kubernetes spec files and adds flagd and associated objects to the workloads based on annotations and custom resource definitions it understands.
+
+## Let's do it!
+
+### Prerequisites
+
+- If you don't have access to an existing K8s cluster, we recommend you install [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) for this tutorial. _kind_ is is similar to minikube (another solution for running a cluster locally you may be familiar with) but supports more than one node, so it makes for a slightly more realistic experience.
+  - If using kind, you'll also need this cluster spec file: [kind-cluster.yaml](@site/static/samples/kind-cluster.yaml), which defines a 3-node cluster with an forwarded containerPort
+- the file defining our demo deployment, service, and CRD: [end-to-end.yaml](https://raw.githubusercontent.com/open-feature/playground/main/config/k8s/end-to-end.yaml) (more on this later).
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [k9s](https://k9scli.io/) (optional, if you'd like to inspect your cluster visually)
+
+### Let's see the code!
```

Comment:
I think this isn't necessary for a first-time tutorial. 
Effectively we are teaching them the cause and effect of editing the CR and seeing it in the web browser.

Reply:
That's how I feel as well.